### ISSUE TITLE
[FE] Switch icons to v3

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#71840]
## Description
Convert the icons we are using with font-awesome to use the new icon components: https://design.va.gov/storybook/?path=/docs/uswds-va-icon--default

The building should be swapped to the `location_city` icon and the road will use the `directions` icon.

## Tasks
- [ ] Update building to be location_city
- [ ] Update road to be directions
- [ ] [FE] Switch icons to v3
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations (readme and use cases)

## Acceptance Criteria
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Tested on staging
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature
- [ ] Prod validation:
  - [ ] not necessary
  - [ ] scheduled

## Code Review Steps
- [ ] Read ticket details and verify that ACs are met
- [ ] Review code in PR
- [ ] Run branch locally
- [ ] Check that new and updated translations use new keys
- [ ] Check for hardcoded strings

**Comment by loripusey** [comment]
waiting on spike ticket before pointing

**Comment by lee-delarm6** [comment]
Ticket for blocker on using v3 added today

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2353

**Comment by loripusey** [comment]
no longer needed
